Lib/test/test_abc.py

```diff
@@ -411,7 +411,35 @@ class MyInt(int):
             self.assertIsInstance(42, A)
             self.assertIsInstance(42, (A,))
 
-        def test_issubclass_bad_arguments(self):
+        def test_subclasses(self):
```

Comment:
Considering `__mro__` is also tested, we should exercise the paths when `__mro__` does not return a tuple.